CMakeLists.txt

```diff
@@ -334,3 +335,105 @@ conditional_directory(AF_BUILD_EXAMPLES examples)
 conditional_directory(AF_BUILD_DOCS docs)
 
 include(CPackConfig)
+
+set_property(GLOBAL APPEND PROPERTY FeatureSummary_PKG_TYPES CPU)
```

Comment:
I am not sure how `FeatureSummary_PKG_TYPES` affects the summary outlook, but I believe this is introduced in 3.8 CMake and that would bump up and minimum requirement to 3.8 from 3.5. This may not fail on CI systems as we don't have 3.5 cmake anywhere at the moment.

Reply:
Either that or CMake devs forget to document this variable in 3.5 docs :)

Reply:
Yes, I just looked at the docs too and it looks like it was introduced in 3.8 instead. As umar suggested, I could disable the feature summary for older versions of cmake. For now though I'm going to close this PR to work on more urgent tasks, and I'll get back to this in the future. Thanks for reviewing!